sdk/scheduler/src/main/java/org/apache/mesos/specification/ContainerSpec.java

```diff
@@ -3,11 +3,16 @@
 import com.fasterxml.jackson.annotation.JsonProperty;
 import com.fasterxml.jackson.databind.annotation.JsonDeserialize;
 
+import java.util.Optional;
+
 /**
  * Spec for defining a Container.
  */
 @JsonDeserialize(as = DefaultContainerSpec.class)
 public interface ContainerSpec {
     @JsonProperty("image-name")
```

Comment:
Not part of this PR, but `image-name` has a hyphen and `rlimit_spec` has an underscore.